client/multi-currency/enabled-currencies-list/list-item.js

```diff
@@ -12,15 +12,32 @@ import { Button, Icon } from '@wordpress/components';
 import DeleteButton from './delete-button';
 
 const EnabledCurrenciesListItem = ( {
+	// eslint-disable-next-line camelcase
```

Comment:
Are these eslint rules needed here and on line 32? I removed them locally and am not seeing any issues.

Reply:
Good catch! I think that these lines were updated as part of the PR to introduce Psalm to the Client (https://github.com/Automattic/woocommerce-payments/pull/2337) and this change made these eslint lines unnecessary. Will remove them and update the PR! :)

Reply:
Fixed in 1f9588f.